apps/api/src/controllers/v1/map.ts

```diff
@@ -202,10 +199,10 @@ export async function getMapResults({
     twoDaysAgo.setDate(twoDaysAgo.getDate() - 2);
 
     // If sitemap is not ignored and either we have few URLs (<100) or the data is stale (>2 days old), fetch fresh sitemap
+    // TODO: We should always soft-fetch the sitemaps from just the index, which fails very fast if there's no result. - mogery
     if (
       !ignoreSitemap &&
-      (sitemapIndexResult.urls.length < 100 ||
-        new Date(sitemapIndexResult.lastUpdated) < twoDaysAgo)
+      (indexResults.length < 100)
```

Comment:
Keep the same functionality as it was before

Reply:
done